app/core/Tracker2/index.js

```diff
@@ -41,19 +47,19 @@ export default class Tracker2 extends BaseTracker {
 
   async _initializeTracker () {
     try {
-      await Promise.all([
-        this.cookieConsentTracker.initialize(),
-
-        ...this.trackers.map(t => t.initialize())
-      ])
+      const allTrackers = [
+        this.cookieConsentTracker,
+        ...this.trackers
+      ]
 
-      this.onInitializeSuccess()
+      await allSettled(allTrackers.map(t => t.initialize()))
     } catch (e) {
-      this.onInitializeFail(e)
+      console.error('Tracker init failed', e)
+    } finally {
+      // We always allow the master tracker to continue because some sub trackers may have initialized correctly
+      this.onInitializeSuccess()
```

Comment:
Would we ever want this to fail if everything has failed? Or does this need to be called to allow sign up to succeed?

Reply:
I suppose we could check the fail state of all of the trackers and then fail the larger tracker call but I don't see much of an advantage of this right now as the lower level trackers will also fail themselves.  This line ensures that we always try our sub trackers.  What do you think?